changelog/2283.removed.md

```diff
@@ -0,0 +1 @@
+The `LinkatFlags` type has been deprecated, please use `AtFlags` instead.
```

Comment:
These file names are wrong.  They should have 2284 in the name, not 2283.

Reply:
Thanks for catching it!